MainTel/MainTel-Non-standard-Operations/MTM2-Non-standard-Procedures/Power-cycle-MTM2-cabinet.rst

```diff
@@ -0,0 +1,174 @@
+
+  - You cannot ping or ssh the M2 controller  m2-crio-controller01.cp.lsst.org (on TMA) **m2-crio-controller02.cp.lsst.org** (on level 3).
+
+  - You see errors codes 6051 and 6088, or 6051 and 6052, and in some cases error 6056.
+
+- Know the location of the MTM2 cabinet according its installation, which is either in TMA or on level 3.
+
+.. _Post-Condition:
+
+Post-Condition
+==============
+
+.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Please provide screenshots of the software status or relevant display windows to confirm.
+.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
+
+- MTMM2 GUI displays no alarms, and MTM2 can transition to a close-loop status.
+
+
+.. _Procedure-Steps:
+
+Procedure Steps
+===============
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+Depending on the location of MTM2 the corresponding steps are:
+
+.. _If-MTM2-is-on-the-TMA:
+
+If MTM2 is on the TMA
+-----------------------
+
+
+.. .. _`tea-pdu`: https://tea-pdu01.cp.lsst.org/ this link currently (May 30, 2024) get a "This site can't be reached"
+
+#. Enter **tea-pdu**, this is the internet interface of the general power distribution (PDU). 
+
+    - You can see a screenshot in :ref:`pdu-m2` figure below.
+
+#. Find the credentials in the **Operators Vault**, under the name PDU Utilities Cabinet. *Note*: The link to tea-pdu address is in this item.
+
+#. Select **outlets** in the left panel of the window.
+
+#. Select and power :guilabel:`Off` M2 Cabinet.
+
+#. Wait 30 seconds.
+
+#. Turn the power :guilabel:`On` the M2 Cabinet.
+
+.. _pdu-m2:
+
+.. figure:: /MainTel/MainTel-Non-standard-Operations/MTM2-Non-standard-Procedures/_static/pdu_m2.png
+    :name: Outlet section at tea-pdu
+
+.. _If-MTM2-is-on-level-3:
+
+If MTM2 is on level 3
+----------------------
+
+#. Go to level 3.
+
+#. Disconnect the power cable.
```

Comment:
Where is the power cable? Do we have any more instructions for this cable?

Reply:
No, I don't know where it is, I asked Kevin. Te-Wei suggested a photo of the cable aswell.

Reply:
There was some concern for a safety hazard in OS team, but Franco informed me that power cycle is done by pushing a button now, so it is now safe. I'll update the document next week as Franco will give me instructions and provide a photo.